### ISSUE TITLE
doclayout_yolo_docstructbench_imgsz1280_2501.pt

### ISSUE THREAD

**Issue by toupride-moona** [opendatalab/DocLayout-YOLO#193]
doclayout_yolo_docstructbench_imgsz1280_2501.pt 是否支持微调，测试发现如果微调，原来能力消失啦

**Comment by toupride-moona** [comment]
for name, param in model.named_parameters(): param.requires_grad 都为False

**Comment by JulioZhao97** [comment]
@toupride-moona 您好，能具体讲一下在什么样子的数据上微调的吗，还有具体的setting

**Comment by toupride-moona** [comment]
> [@toupride-moona](https://github.com/toupride-moona) 您好，能具体讲一下在什么样子的数据上微调的吗，还有具体的setting

我是因为识别出来的数据识别为丢弃的，所以微调，label-studio标注少量数据。使用doclayout_yolo_docstructbench_imgsz1280_2501.pt。类似(https://blog.csdn.net/2301_80224467/article/details/151581176)，但是我发现doclayout_yolo_docstructbench_imgsz1280_2501.pt的 for name, param in model.named_parameters(): param.requires_grad 都为False ，所以都已经冻住了。文章说的也不对。

**Comment by JulioZhao97** [comment]
@toupride-moona 具体用了多少数据？使用的数据其他类别的标注是完整的吗，还是只有这几个丢弃的类别

**Comment by toupride-moona** [comment]
> [@toupride-moona](https://github.com/toupride-moona) 具体用了多少数据？使用的数据其他类别的标注是完整的吗，还是只有这几个丢弃的类别

数据很少，几十个，类别是完整的。
